tests/unit/selectmenu/selectmenu_methods.js

```diff
@@ -78,21 +78,21 @@ asyncTest( "refresh - change selected option", function() {
 	var element = $( "#speed" ).selectmenu(),
 		button = element.selectmenu( "widget" );
 
-	equal( element.find( "option:selected" ).text(), button.text(), "button text after init" );
+	equal( " " + element.find( "option:selected" ).text(), button.text(), "button text after init" );
 
 	button.simulate( "focus" );
 	setTimeout(function() {
-		equal( element.find( "option:selected" ).text(), button.text(), "button text after focus" );
+		equal( " " + element.find( "option:selected" ).text(), button.text(), "button text after focus" );
 
 		element[ 0 ].selectedIndex = 0;
 		element.selectmenu( "refresh" );
-		equal( element.find( "option:selected" ).text(), button.text(),
+		equal( " " + element.find( "option:selected" ).text(), button.text(),
 			"button text after changing selected option" );
 
 		element.find( "option" ).prop( "selected", false );
 		element.append( "<option selected value=\"selected_option\">Selected option</option>" );
 		element.selectmenu( "refresh" );
-		equal( "Selected option", button.text(), "button text after adding selected option" );
+		equal( " Selected option", button.text(), "button text after adding selected option" );
```

Comment:
Bad argument order, let's fix this while we're here? Should be [actual][expected][message].